python/MuonPhotonMatch_cff_miniAOD.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+# JMTBAD muon-photon matching is done here using the default muon
+# momentum and not whichever refit momentum will be eventually used in
+# the analysis. For most muons this shouldn't make much difference as
+# the matching is in eta-phi, but if the default is really bad and the
+# selected refit manages to recover, the photon match may be
+# wrong. The photon brem recovery needs to be studied more anyway.
+muonPhotonMatch = cms.EDProducer('TrivialDeltaRViewMatcher',
+                                 src     = cms.InputTag('slimmedMuons'),
+                                 matched = cms.InputTag('slimmedPhotons'),
+                                 distMin = cms.double(0.1)
+                                 )
+
+def addUserData(patMuonProducer, tag=cms.InputTag('muonPhotonMatch')):
+    patMuonProducer.userData.userCands.src.append(tag)
```

Comment:
There shouldn't be any confusion with it unless both muonPhotonMatch_cff_miniAOD.py and MuonPhotonMatch_cff.py are both imported for some reason, which does not happen in any of the modifications I've made. 

I think it does make sense from a logical standpoint though, and I don't think it will be difficult to change. I will try to implement it and see if there are any issues.